src/actions/upload.js

```diff
@@ -87,6 +87,16 @@ const upload = (packageData, user) => {
     })
 }
 
+const checkUploadFileSize = packageData => {
+  const stats = fs.statSync(packageData.tgzFile)
+  const fileSizeInMB = stats.size / 1000000 //convert from Bytes to MB
+
+  if (fileSizeInMB > 10) {
```

Comment:
hey @sandeep-vedam, can we make this `>=` (just to be sure)

Reply:
Hey @michielvandergeest ,

I have updated as per the comment.